### PR TITLE
Do not limit the set of architectures to build on

### PR DESCRIPTION
**Pull request by fedelibre**
As org.freedesktop.Sdk.Extension.texlive is now built from
source, I can build for any architecture including aarch64.

### CODE REVIEW

**Comment by hfiguiere** [comment]
You initial build issue is that the tarball is too old for aarch64 and needs updated `config.guess` and `config.sub`

The way to do so is to copy them from the local install in the sdk. This is done by adding a source entry to the module:
```json
        {
          "type": "shell",
          "commands": [
            "cp -p /usr/share/automake-*/config.{sub,guess} ."
          ]
        }
```

(may need to be tweaked depending on where the files are in the tarball, this works in most cases though)

**Comment by hfiguiere** [comment]
Commit cfef69b doesn't help and should be removed (to be clear)

**Comment by fedelibre** [comment]
Thanks. These files are in the build-aux/ directory in the tarball and default permission is read-only. Changing permission with chmod and then using your cp command seem to work fine here on my laptop (x86_64).

I'll push and test the aarch64 build soon.

File: org.frescobaldi.Frescobaldi.yaml [hunk 1]
```diff
@@ -45,6 +45,11 @@ modules:
         path: patches/guile-1.8.8-cve-2016-8605.patch
       - type: patch
         path: patches/guile-1.8.8-configure.patch
+      # Files in the tarball are too old for aarch64 so we take them from the Sdk.
+      - type: shell
+        commands:
+          - chmod +w build-aux/config*
+          - cp -p /usr/share/automake-*/config.{sub,guess} build-aux/
```
Comment (hfiguiere) [inline org.frescobaldi.Frescobaldi.yaml:52]: there is also `guile-readline`.

Also `cp -f` is enough to overwrite the permissions.

**Comment by fedelibre** [comment]
Now ghostscript is failing. I'll copy from [Inkscape manifest](https://github.com/flathub/org.inkscape.Inkscape/blob/master/org.inkscape.Inkscape.json#L246).

**Comment by fedelibre** [comment]
Before trying to work around the Pyqt error... I don't have many ideas to test, except the qmake path maybe.

The problem is I cannot build anymore even x86_64 on my laptop. guile module is failing, see [guile-error.txt](https://github.com/flathub/org.frescobaldi.Frescobaldi/files/5906981/guile-error.txt).

I'm curious to know why x86_64 build works here and it doesn't (anymore) on my computer.
I did a fresh build.

**Comment by fedelibre** [comment]
Replaced by #3

**Pull request closed without merging**